technical_guides/chef.md

```diff
@@ -44,7 +44,7 @@
 More Information / References
 =============================
 
-  * [The Berkshelf Way ~ Jamie Winsor](http://www.getchef.com/blog/chefconf-talks/the-berkshelf-way-jamie-winsor/)
+  * The Berkshelf Way ~ Jamie Winsor, formerly at: http://www.getchef.com/blog/chefconf-talks/the-berkshelf-way-jamie-winsor/
```

Comment:
I found only one link to something that matched and it was on a site so laden in ads that safari struggled to load the next page.  I've decided I'd rather not link to them.

Reply:
I'd suggest we drop it. The video seems to not exist on the internet anymore, and the slides are pretty useless without the narration.

Reply:
done